templates/_helpers.tpl

```diff
@@ -61,18 +61,9 @@ Define the http port of the Ambassador service
 {{- end -}}
 
 {{/*
-Filter the valid deploymentTool
+Return a valid managed-by. By default "Helm", but can be overriden with `deploymentTool`
 */}}
 {{- define "ambassador.managedBy" -}}
-    {{- if .Values.deploymentTool -}}
```

Comment:
Rather than removing the filters, should we add checks for the additional values we now consider valid?

Reply:
I think it is not worth it.